playbooks/mysql_staging.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: installs mysql on staging
+  hosts: mysql-db-staging1.princeton.edu
+  remote_user: pulsys
+  become: true
+  vars_files:
+    - ../group_vars/mysql/mysql_staging.yml
+    - ../group_vars/mysql/vault.yml
+  pre_tasks:
+    - name: Upgrade all apt packages
+      apt:
+        upgrade: dist
+        force_apt_get: true
+
+    - name: install pip on new VM
+      raw: test -e /usr/bin/python
+      changed_when: false
+      failed_when: false
+      register: check_python
+
+    - name: Install Python  # this has to be raw in the event python is not installed 
```

Comment:
Again, if no python is installed on a box at all, I would expect the first pre-task to fail. If we need to handle the case where a new VM has no python installed at all, we probably need to do that first. If we're just looking to get a more recent python, or python in the right place, I think we can use `apt`.